code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -157,10 +157,10 @@
 	required_reagents = list(/datum/reagent/water = 1, /datum/reagent/silicon = 1, /datum/reagent/acetone = 1)
 	result_amount = 4
 
-/datum/chemical_reaction/pacid
-	name = "Polytrinic acid"
-	result = /datum/reagent/acid/polyacid
-	required_reagents = list(/datum/reagent/acid = 1, /datum/reagent/acid/hydrochloric = 1, /datum/reagent/potassium = 1)
+/datum/chemical_reaction/tacid
+	name = "Triflic acid"
+	result = /datum/reagent/acid/triflicacid
+	required_reagents = list(/datum/reagent/toxin/potassium_chlorophoride = 1, /datum/reagent/acid/hydrochloric = 1, /datum/reagent/lexorin = 1)
```

Comment:
Renaming the reagent breaks the recipes where it's used, mainly **metal foam and nitroglycerin**, metal foam nades are actually used commonly and I'm pretty sure this will cause issues when metalfoam grenades call for polyacid, if they use actual chemicals in their code.

Reply:
Oops. I'll have to fix that.